.github/workflows/deploy.yml

```diff
@@ -0,0 +1,92 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/deploy.yml
+      - README.md
+      - acknowledgements.html
+      - biblio.js
+      - terms.html
+      - acknowledgements/**
+      - css/**
+      - img/**
+      - script/**
+      - utility/**
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == "w3c/aria-common" }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - repo: accname
+            branch: main
+          - repo: aria
+            branch: main
```

Comment:
I didn't try to see if the `branch` could also just be an array in the matrix for the few that had multiple values for the same repo